sky/backends/backend_utils.py

```diff
@@ -1756,7 +1809,8 @@ def refresh_cluster_status_handle(
 
     handle = record['handle']
     if isinstance(handle, backends.CloudVmRayBackend.ResourceHandle):
-        if force_refresh or record['autostop'] >= 0:
+        use_spot = handle.launched_resources.use_spot
+        if (force_refresh or record['autostop'] >= 0 or use_spot):
             # Refresh the status only when force_refresh is True or the cluster
             # has autostopped turned on.
```

Comment:
Remove? Since it's textual description of the conditions.

Reply:
Good point. Removed. Thanks!